src/company.js

```diff
@@ -17,41 +25,17 @@ export const CompanyList = () => (
             <TextField source="country" />
             <DateField source="createdAt" />
             <DateField source="modifiedAt" />
+            <EditButton variant="outlined" />
         </Datagrid>
     </List>
 );
 
 
 export const CompanyEdit = () => (
-    <Edit>
-        <SimpleForm>
-            <TextInput disabled source="id" />
-            <ReferenceInput source="tenantId" reference="tenants">
-                <SelectInput optionText="displayName" />
-            </ReferenceInput>
-            <TextInput source="name" />
-            <TextInput source="address" />
-            <TextInput source="zipCode" />
-            <TextInput source="city" />
-            <TextInput source="state" />
-            <TextInput source="country" />
-        </SimpleForm>
-    </Edit>
+    <EditEntityLayout inputsList={inputsList} referenceValues={referenceValues} />
 );
 
 
 export const CompanyCreate = () => (
-    <Create>
-        <SimpleForm>
-            <ReferenceInput source="tenantId" reference="tenants">
-                <SelectInput optionText="displayName" />
-            </ReferenceInput>
-            <TextInput source="name" />
-            <TextInput source="address" />
-            <TextInput source="zipCode" />
-            <TextInput source="city" />
-            <TextInput source="state" />
-            <TextInput source="country" />
-        </SimpleForm>
-    </Create>
+    <CreateEntityLayout inputsList={inputsList} referenceValues={referenceValues} />
 );
```

Comment:
Please add a new line at the end of the files

Reply:
Done